pinot-query-runtime/src/main/java/org/apache/pinot/query/mailbox/channel/GrpcMailboxServer.java

```diff
@@ -37,17 +54,33 @@
  * send by the sender of the sender/receiver pair.
  */
 public class GrpcMailboxServer extends PinotMailboxGrpc.PinotMailboxImplBase {
+  private static final Logger LOGGER = LoggerFactory.getLogger(GrpcMailboxServer.class);
```

Comment:
The PR title is aimed at QueryServer. Do you also need this for the GrpcMailboxServer? If so you also need to enable this in `GrpcSendingMailbox`. If you don't need this then you can revert the changes in this file.

Reply:
Removed the GrpcMailboxServer - If required will add as a separate PR